Python/5-partial-ext.py

```diff
@@ -0,0 +1,18 @@
+
+def partial(fn, *args):
+    return lambda *rest: fn(*(args + rest))
+
+
+def summary(a, b, c, d):
```

Comment:
Let's use the same identifacators names so a person learning different languages would have easier time understanding the analogy between JavaScript and Python
```suggestion
def sum4(a, b, c, d):
```